spec/jasmine-helper.js

```diff
@@ -0,0 +1,222 @@
+'use babel'
+/* @flow */
```

Comment:
FYI, here is the issue to track for declaring "all my files are Flow" in a `.flowconfig`: https://github.com/facebook/flow/issues/284